EventProc/python/trigScintClustering.py

```diff
@@ -0,0 +1,50 @@
+"""Configuration for Trigger Scintillator cluster producer 
+
+Sets all parameters to reasonable defaults.
+
+Examples
+--------
+    from LDMX.EventProc.trigScintClustering import TrigScintClusterProducer
+    p.sequence.extend([ TrigScintClusterProducer.up() , TrigScintClusterProducer.down() , TrigScintClusterProducer.tagger() ])
+"""
+
+from LDMX.Framework import ldmxcfg
+
+class TrigScintClusterProducer(ldmxcfg.Producer) :
+    """Configuration for cluster producer for Trigger Scintillators"""
+
+    def __init__(self,name) :
+        super().__init__(name,'ldmx::TrigScintClusterProducer')
+
+        from LDMX.EventProc import include
+        include.library()
```

Comment:
In order to update the container with the cmake changes, I removed the automaic generation of the `include.py` module. I replaced it with a change to the constructor:
```
super().__init__(name,'ldmx::TrigScintClusterProducer','EventProc')
```